src/main/java/seedu/address/logic/commands/ArchiveCommand.java

```diff
@@ -71,9 +71,10 @@ private static InternshipApplication createdArchivedApplication(InternshipApplic
         CompanyName companyName = internshipApplication.getCompanyName();
         JobTitle jobTitle = internshipApplication.getJobTitle();
         Contact contact = internshipApplication.getContact();
+        InternshipStatus status = internshipApplication.getStatus();
         InterviewDate interviewDate = internshipApplication.getInterviewDate();
 
-        return new InternshipApplication(companyName, jobTitle, contact, InternshipStatus.ARCHIVED, interviewDate);
+        return new InternshipApplication(companyName, jobTitle, contact, status, true, interviewDate);
```

Comment:
Btw @benjamin-wee , I think this part may affect your reminder function as well 🤔